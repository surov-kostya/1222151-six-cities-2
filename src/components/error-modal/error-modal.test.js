import React from 'react';
import renderer from 'react-test-renderer';
import ErrorModal from './error-modal';

it(`City list renders correctly`, () => {
  const tree = renderer.create(<ErrorModal errorMessage={`error`}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
