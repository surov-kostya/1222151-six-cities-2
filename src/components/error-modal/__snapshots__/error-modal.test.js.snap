// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City list renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(0,0,0, .7)",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
      "position": "fixed",
      "width": "100vw",
      "zIndex": "1000",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "white",
        "borderRadius": "5px",
        "display": "flex",
        "flexDirection": "column",
        "height": "200px",
        "overflow": "hidden",
        "padding": "10px",
        "width": "300px",
      }
    }
  >
    <p
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "flexGrow": "1",
          "justifyContent": "center",
        }
      }
    >
      error
    </p>
    <button
      style={
        Object {
          "backgroundColor": "#4481c3",
          "border": "0",
          "borderRadius": "3px",
          "color": "white",
          "width": "100px",
        }
      }
    >
      Ok
    </button>
  </div>
</div>
`;
